src/styles/utilities/_functions.scss

```diff
@@ -0,0 +1,7 @@
+@function getFirstDayPosition($box-size, $box-gap, $day-number) {
+  @return ($box-size + $box-gap) * $day-number;
+}
+
+@function getMaxWidth($box-size, $box-gap, $box-paddings) {
+  @return ($box-size * 7) + ($box-gap * 6) + ($box-paddings * 2);
```

Comment:
Not necessarily a mistake, but if you do not set `box-sizing` to `border-box`, but instead keep it a default
`content-box`, you don't need to account for `$box-paddings`

Reply:
But in that case you would also need to set `border-box` for the day boxes